app/cells/decidim/tags/equity.erb

```diff
@@ -1 +1,4 @@
-<%= content_tag :li, (content_tag :span, equity_name, class: "equity_score") %>
+<span class="equity_score"><%= equity_name %></span>
+|
+<a href="<%= proposal_path %>"><%= t("card_more_info", scope: "decidim.proposals.equity") %></a>
```

Comment:
Should the `What is this?` link go somewhere besides just the proposal? For some reason I thought this was going to link to an external page, but I could be wrong.

Reply:
Good question. The intent from Josh is to have a tooltip open up, but we didn't have time to add one... I chose to link to the proposal page for now because we have some explanation there. A goal was to show that there is an education element to this UI even though it's non-final.

Another option I concerned was to make the link to nothing, to show that it isn't done, but I felt that'd be pretty confusing for Shaun and other stakeholders